src/Core/composer.json

```diff
@@ -18,7 +18,7 @@
         "psr/cache": "^1.0",
         "psr/log": "^1.0",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/http-client": "^4.4 || ^5.0, !=5.2.0",
+        "symfony/http-client": "^4.4 || ^5.0,!=5.2.0",
```

Comment:
For consistency with spaces arround ` || `.

Reply:
That will fail `composer normalize`

https://github.com/async-aws/aws/pull/896/checks?check_run_id=1531089092#step:4:31

Reply:
oh... ok then